backend/src/main.ts

```diff
@@ -3,6 +3,12 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  app.enableCors({
+    origin: process.env.FRONTEND_URL,
+    credentials: true,
+  });
+
   await app.listen(3000);
```

Comment:
Maybe just use port 8787 as with the other PR to keep consistency sake. This also can't be on port `3000` as it conflicts with nextjs.
```suggestion
  await app.listen(8787);
```